extensions/email/src/main/java/org/mule/extension/email/api/retriever/matcher/TimeSinceFunction.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.api.retriever.matcher;
+
+import java.time.LocalDateTime;
+import java.util.function.BiFunction;
+
+/**
+ * A Boolean {@link BiFunction} which returns {@code true} if a given
+ * {@link LocalDateTime} is posterior or equal to a reference date
+ *
+ * @since 4.0
+ */
+public final class TimeSinceFunction implements BiFunction<LocalDateTime, LocalDateTime, Boolean>
```

Comment:
isn't this exactly the same as the one file has? if that's the case, let's move this to an api.util package in the core